components/Orders/ProposalOpenOrderRow.tsx

```diff
@@ -56,7 +56,7 @@ export function ProposalOpenOrderRow({ order }: { order: OpenOrdersAccountWithKe
 
     try {
       setIsCanceling(true);
-      await cancelOrder(order, marketAccount.publicKey);
+      await cancelAndSettleOrder(order, marketAccount.publicKey);
       await fetchBalance(marketAccount.account.baseMint);
```

Comment:
Do we want these, or can we wrap them in a check against WS connected?

Reply:
I think we can drop this fetch balance now since we should always be listening to updates on the user's META token ATA.

Reply:
Yea it's either we catch it and move these things into hooks so they make more sense where and when they're used.. Cause this happens in a lot of places and should likely be managed through some logic either in the context or wherever, but have that managed for all external processes with the RPC, then we're finally approaching some sense.